homeassistant/components/logbook.py

```diff
@@ -37,19 +39,19 @@
 
 ATTR_NAME = 'name'
 ATTR_MESSAGE = 'message'
-ATTR_DOMAIN = 'domain'
+ATTR_DOMAIN_INPUT = 'component'
```

Comment:
Problem is this in ```call```:
```
        call_id = self._generate_unique_id()
        event_data = service_data or {}
        event_data[ATTR_DOMAIN] = domain
        event_data[ATTR_SERVICE] = service
        event_data[ATTR_SERVICE_CALL_ID] = call_id
```

Ill push so you can see the problem, domain is always None